src/jackett.py

```diff
@@ -113,30 +118,35 @@ def filter_results(method, results):
 
     if get_setting('filter_keywords_enabled', bool):
         results = filter.keywords(results)
-        log.debug(f"results after filtering keywords: {results}")
+        log.info(f"filtering keywords {len(results)}")
+        log.debug(f"filtering keywords results: {results}")
```

Comment:
All of these additional log entries are fine, but it makes more sense to add it before the call, and then have the debug after the call, so in this case for example it would be something like

```
log.info(f"filtering keywords")
results = filter.keywords(results)
log.debug(f"now at {len(results)} results: {results}")
```

Reply:
done